### PR TITLE
Misc doc updates

### PR DESCRIPTION
**Pull request by Samq64**
- fixes a typo @Jazza-231 reported
- documents `untranslated`
- Turns `type` into a list
- Recommend Firefox for Android in the FAQ leaving Kiwi as an alternative
- Use `e.key` instead of `keyCode`

### CODE REVIEW

File: content/docs/reference/addon-manifest.md
Comment (WorldLanguages) [inline content/docs/reference/addon-manifest.md:371]: I thought "untranslated" was a setting type, not a property.

**Review by Hans5958 — APPROVED** [review]
Tacit approval since you asked (aka. you don't need my approval, and should wait for other reviews)

File: content/docs/develop/userscripts/best-practices.md
Comment (WorldLanguages) [inline content/docs/develop/userscripts/best-practices.md:114]:
```suggestion
- When listening to keyboard events, accessing `event.key` the preferred way to know which key was pressed. In general, you should avoid `event.key` and `event.keyCode`.
```

File: content/docs/reference/addon-manifest.md
Comment (WorldLanguages) [inline content/docs/reference/addon-manifest.md:368]: Would be a nice opportunity to turn this into an item list.

I think the best way to describe the untranslated type is:
> Works as a string setting, but its default value is not sent to the extension translation process. For example, if the default value is a URL or a number.

Comment (Samq64) [reply content/docs/reference/addon-manifest.md:368]: I started trying to turn it into a table locally but I'm not really sure exactly how to lay it out.

**Comment by WorldLanguages** [comment]
Thanks!

**Pull request merged**
